core/src/modules/errors.ts

```diff
@@ -1,8 +1,19 @@
+import { WhereOptions } from "sequelize/types";
+
 export namespace Errors {
   export const ERROR_LEVELS = ["error", "info"] as const;
   export type ErrorLevel = typeof ERROR_LEVELS[number];
 
-  export class InfoError extends Error {
+  export class GrouparooError extends Error {
```

Comment:
The properties defined here are in part based on the usage [here](https://github.com/grouparoo/grouparoo/blob/62de8fde7fbcf658ba30e22d4b23d1d45e5444f4/core/src/config/errors.ts#L147)